### DISCUSSION TITLE
Self-nominate @blakerosenthal to Contributors team

### DISCUSSION THREAD

**Discussion by blakerosenthal** [nebari-dev/nebari#2750] [Community]
Hi folks, I'd like to nominate myself to join the Nebari contributors team! I have made [a few contributions so far](https://github.com/nebari-dev/nebari/commits/develop/?author=blakerosenthal) and will be able to ramp up even more over the remainder of the year. There is [at least one issue](https://github.com/nebari-dev/nebari/issues/2701) I'd like to work on that requires elevated permissions to trigger GHA workflows, so that's the main reason for this request.

**Comment by Adam-D-Lewis** [comment]
I support this

**Comment by marcelovilla** [comment]
Me too 🚀

**Comment by viniciusdc** [comment]
wonderful @blakerosenthal! Supported

**Comment by dcmcand** [comment]
@dharhas can you add @blakerosenthal to the contributers team?

**Comment by dharhas** [comment]
done
